### DISCUSSION TITLE
BUG: Push is stuck at "processing update" in PR

### DISCUSSION THREAD

**Discussion by AkashMehta27** [community/community#118716] [Pull Requests]
### Select Topic Area

Bug

### Body

I had an open PR and pushed some changes to it.

The new changes didn't show up instead it said "processing update". 

I refreshed the browser a couple of times and that message didn't go away. 

I tried to edit the changes file directly in the PR, but it showed the new version on the edit page so I canceled. The "processing update" message still came back.

At this point, I closed the PR and opened a new one instead, and this bug was gone.

I can't really share any images of this since I am unable to see that error in any of my other PRs

System: Windows
Browser: Edge

**Comment by whittede** [comment]
Encountered this same issue today. Had an open PR that I needed to make some changes on because of some review items pointed out by my team. In order to make those changes, I pulled in the changes that had since been made on the main branch after my branch had been created. (I think ~75 commits ahead.) I then added two commits of my own.

I then pushed up all my commits into the PR. No problems. Git says the changes went through fine and that I'm up-to-date with nothing new to push. However, for the last eight hours GitHub has been stuck saying "Processing Updates" and doesn't display my commits in the PR.
![Console showing no errors](https://github.com/user-attachments/assets/505710a4-eac9-4aba-896f-a9f3aa91ace7)
![GitHub permanently stuck](https://github.com/user-attachments/assets/5723c52b-44f9-479a-822e-c9ef47e90981)

**Comment by m3di** [comment]
it happened to me just now.
it's stuck for almost 30 minutes.
what happened after 8 hours?

**Comment by m3di** [comment]
For anyone visiting this thread from the future:
✔ Restart IDE
✔ Restart PC
✔ Restart Wifi
❌ None of these work

Solution:
Add a random commit like:
fix: convince GitHub to actually update

Suddenly… it works.

**Comment by luvl** [comment]
<img width="988" height="110" alt="image" src="https://github.com/user-attachments/assets/ac6d4427-ed76-4a7b-afb4-aab2b6ef7b1a" />
Add a empty commit and it worked

`git commit --allow-empty -m "chore: trigger PR update" && git push`

**Reply by Rini-Tikki** [reply]
adding an empty comment is also not working for me.

**Reply by fuad1502** [reply]
It "kind of" works for me. After pushing the empty commit, the previous commit is now listed in the PR, but the PR still shows "Processing updates".

**Reply by Hellionfrs** [reply]
This worked for me 💡

**Reply by therealharshit** [reply]
Not working for me

**Reply by ryan-farrell** [reply]
This worked for me thanks 👍🏼

**Reply by SlimJ4** [reply]
Worked for me too. Awesome. Since three of us are commenting here in the span of a few minutes, I suppose there's an issue on the end of GitHub ?

EDIT : Swing and miss, it went back to pending for some reason.

**Reply by FedericaMelani** [reply]
yeap there is an issue on Github https://www.githubstatus.com/ probably related

**Reply by therealharshit** [reply]
I had to do a force push btw

**Reply by Djirasak** [reply]
This worked for me thanks 👍🏼

**Reply by Ichi1234** [reply]
This worked for me too. My PR is in this state for around 20 hours. 😅
